workers/run_tests.sh

```diff
@@ -190,55 +223,69 @@ if [ -z "$tag" ] || [ "$tag" = "agilent" ]; then
         wget -q -O "$no_test_raw_dir/$no_file2" \
              "$test_data_repo/$no_file2"
     fi
-    no_file3="GSM557500_sample_table.txt"
+    no_file3="GSM1234847_sample_table_headerless.txt"
     if [ ! -e "$no_test_raw_dir/$no_file3" ]; then
         mkdir -p "$no_test_raw_dir"
-        echo "Downloading NOOP file3."
-        wget -q -O "$no_test_raw_dir/$no_file3" \
-             "$test_data_repo/$no_file3"
+        echo "Processing NOOP file3."
+        tail -n +2 "$no_test_raw_dir/$no_file2" > "$no_test_raw_dir/$no_file3"
     fi
-    no_file4="GSM269747-tbl-1.txt"
+    no_file4="GSM1089291-tbl-1.txt"
     if [ ! -e "$no_test_raw_dir/$no_file4" ]; then
         mkdir -p "$no_test_raw_dir"
         echo "Downloading NOOP file4."
         wget -q -O "$no_test_raw_dir/$no_file4" \
              "$test_data_repo/$no_file4"
     fi
-    no_file5="GSM1234847_sample_table.txt"
+    no_file5="GSM1089291-tbl-1-modified.txt"
     if [ ! -e "$no_test_raw_dir/$no_file5" ]; then
         mkdir -p "$no_test_raw_dir"
         echo "Downloading NOOP file5."
         wget -q -O "$no_test_raw_dir/$no_file5" \
              "$test_data_repo/$no_file5"
     fi
-    no_file6="GSM1089291-tbl-1.txt"
-    if [ ! -e "$no_test_raw_dir/$no_file6" ]; then
-        mkdir -p "$no_test_raw_dir"
-        echo "Downloading NOOP file6."
-        wget -q -O "$no_test_raw_dir/$no_file6" \
-             "$test_data_repo/$no_file6"
-    fi
-    no_file7="GSM1089291-tbl-1-modified.txt"
-    if [ ! -e "$no_test_raw_dir/$no_file7" ]; then
-        mkdir -p "$no_test_raw_dir"
-        echo "Downloading NOOP file7."
-        wget -q -O "$no_test_raw_dir/$no_file7" \
-             "$test_data_repo/$no_file7"
+    no_test_exp_dir="$volume_directory/TEST/NO_OP/EXPECTED"
+    no_exp_file1="gene_converted_GSM557500-tbl-1.txt"
+    if [ ! -e "$no_test_exp_dir/$no_exp_file1" ]; then
+        mkdir -p "$no_test_exp_dir"
+        echo "Downloading NOOP expected file1."
+        wget -q -O "$no_test_exp_dir/$no_exp_file1" \
+             "$test_data_repo/$no_exp_file1"
+    fi
+    no_exp_file2="GSM269747.PCL"
+    if [ ! -e "$no_test_exp_dir/$no_exp_file2" ]; then
+        mkdir -p "$no_test_exp_dir"
+        echo "Downloading NOOP expected file2."
+        wget -q -O "$no_test_exp_dir/$no_exp_file2" \
+             "$test_data_repo/$no_exp_file2"
+    fi
+    no_exp_file3="gene_converted_GSM1234847-tbl-1.txt"
+    if [ ! -e "$no_test_exp_dir/$no_exp_file3" ]; then
+        mkdir -p "$no_test_exp_dir"
+        echo "Downloading NOOP expected file3."
+        wget -q -O "$no_test_exp_dir/$no_exp_file3" \
+             "$test_data_repo/$no_exp_file3"
+    fi
+    no_exp_file4="gene_converted_GSM1089291-tbl-1.txt"
+    if [ ! -e "$no_test_exp_dir/$no_exp_file4" ]; then
+        mkdir -p "$no_test_exp_dir"
+        echo "Downloading NOOP expected file4."
+        wget -q -O "$no_test_exp_dir/$no_exp_file4" \
+             "$test_data_repo/$no_exp_file4"
```

Comment:
Could this just be done as a loop over `"gene_converted_GSM557500-tbl-1.txt GSM269747.PCL gene_converted_GSM1234847-tbl-1.txt gene_converted_GSM1089291-tbl-1.txt"`?